### ISSUE TITLE
Employer /Two-Factor Authentication

### ISSUE THREAD

**Issue by Jamalatm** [Anupriyabanger/Bee-Recruiter#1254]
**Title:**
Error: "Two Factor Authetication"
Correction: "Two-Factor Authentication"
Bolding or increase the font size of the title "Two-Factor Authentication" to make it more prominent and draw immediate attention.

**Description:**
Error: "...enabling Two Factor Authentication."
Correction: "...enabling Two-Factor Authentication."
**Main Content:**
Error: "For example, a website or application might require users to enter For example, a website or application might require..."
Correction: Remove the repetitive phrase. "For example, a website or application might require users to enter their phone (something they have) in order to log in."
Break the paragraph into smaller, more digestible chunks. Long paragraphs can be hard to follow.
add examples of the "three categories" mentioned in the description. This will make it clearer to the user what these categories are.
User Interface:
add an "info" icon or tooltip next to the "Two-Step Verification" button. This can provide a quick overview or definition of what two-step verification is, helping users who may not be familiar with the term.
Ensure consistent spacing between sections for a cleaner look.
add icons or visual cues next to points like "something the user knows," "something the user has," and "something the user is" to visually represent these concepts.

<img width="960" alt="2023-10-24" src="https://github.com/Anupriyabanger/Bee-Recruiter/assets/133124176/2da3443f-9697-4ee8-aa54-7f3a3c766b3c">

**Comment by vinayak-vayuz** [comment]
<img width="960" alt="image" src="https://github@SaurabhattJi m/Anupriyabanger/Bee-Recruiter/assets/128778447/b8adf566-afbd-4d34-ae71-6829d05b0dec">
Done from my end rest @SaurabhattJi will add tooltip

**Comment by SaurabhattJi** [comment]
![image](https://github.com/Anupriyabanger/Bee-Recruiter/assets/120307815/de94a74a-f105-4410-8cb7-9909b452d6df)
Done from my end.

**Comment by Jamalatm** [comment]
@SaurabhattJi  @vinayak-vayuz  Thanks
